roles/matrix-jitsi/templates/web/matrix-jitsi-web.service.j2

```diff
@@ -23,6 +23,7 @@ ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-jitsi-web \
 			{% endif %}
 			--mount type=bind,src={{ matrix_jitsi_web_config_path }},dst=/config \
 			--mount type=bind,src={{ matrix_jitsi_web_transcripts_path }},dst=/usr/share/jitsi-meet/transcripts \
+			--mount type=bind,src={{ matrix_jitsi_web_crontabs_path }},dst=/var/spool/cron/crontabs \
```

Comment:
What creates this crontabs file? Perhaps something is missing from `tasks/setup_install.yml`?

Reply:
I think this is for lets-encrypt renew certs crontab which will be manged by it's docker itself.

Reply:
As far as I know, you can't use `--mount` with a non-existing file (`/matrix/jitsi/web/crontabs` in this case).

Reply:
yes, mount directory should be an valid path. Thanks 👍